src/main/webapp/WEB-INF/view/jsp/default/ui/casLoginView.jsp

```diff
@@ -50,68 +52,124 @@
 				*Please note that loading real patient data into the system is strictly prohibited!
 			</p>
 		</c:if>
-		<div class="row">
-			<div class="col-sm-6 col-sm-offset-3">
-				<form:form method="post" id="fm1" cssClass="" commandName="${commandName}" htmlEscape="true">
-					<div class="form-group has-error">
-						<div class="help-block">
-							<form:errors path="*" cssClass="" id="status" element="span"/>
-						</div>
-					</div>
-					<div class="form-group">
-						<label for="username">Username</label>
-						<form:input cssClass="form-control" cssErrorClass="" id="username" size="25" tabindex="1"
-									accesskey="${userNameAccessKey}" path="username" autocomplete="false"
-									htmlEscape="true"/>
-					</div>
-					<div class="form-group">
-						<label for="password">Password</label>
-							<%--
-							NOTE: Certain browsers will offer the option of caching passwords for a user.  There is a non-standard attribute,
-							"autocomplete" that when set to "off" will tell certain browsers not to prompt to cache credentials.  For more
-							information, see the following web page:
-							http://www.geocities.com/technofundo/tech/web/ie_autocomplete.html
-							--%>
-						<form:password cssClass="form-control" cssErrorClass="" id="password" size="25" tabindex="2"
-									   path="password" accesskey="${passwordAccessKey}" htmlEscape="true"
-									   autocomplete="off"/>
-					</div>
-					<input name="submit" id="submit" class="btn btn-lg btn-primary btn-block" accesskey="l" value="Login"
-						   tabindex="4" type="submit"/>
-					<div style="text-align: right">
-						<a href="${casProperties.applicationUrl}forgot_password.jsp">Forgot Password?</a>
+		<c:choose>
+			<c:when test="${casProperties.localOrLDAPAuthentication}">
```

Comment:
The cas.properties file is being read in by Spring, I believe.

On Tue, Aug 23, 2016 at 10:09 PM, Akshatha Pai <notifications@github.com>
wrote:

> In src/main/webapp/WEB-INF/view/jsp/default/ui/casLoginView.jsp
> <https://github.com/eurekaclinical/cas/pull/13#discussion_r75984622>:
>
> > -							<%--
> > -							NOTE: Certain browsers will offer the option of caching passwords for a user.  There is a non-standard attribute,
> > -							"autocomplete" that when set to "off" will tell certain browsers not to prompt to cache credentials.  For more
> > -							information, see the following web page:
> > -							http://www.geocities.com/technofundo/tech/web/ie_autocomplete.html
> > -							--%>
> > -						<form:password cssClass="form-control" cssErrorClass="" id="password" size="25" tabindex="2"
> > -									   path="password" accesskey="${passwordAccessKey}" htmlEscape="true"
> > -									   autocomplete="off"/>
> > -					</div>
> > -					<input name="submit" id="submit" class="btn btn-lg btn-primary btn-block" accesskey="l" value="Login"
> > -						   tabindex="4" type="submit"/>
> > -					<div style="text-align: right">
> > -						<a href="${casProperties.applicationUrl}forgot_password.jsp">Forgot Password?</a>
> > +		<c:choose>
> > +			<c:when test="${casProperties.localOrLDAPAuthentication}">
>
> i don't see any cases in the codebase where we read the values from the
> .properties file directly from the jsp page using jstl. we call a java
> method to read the value. It is possible to do it as long as we know the
> name of the file. We have the whole logic of which .properties file to use
> in abstract.properties(as in what the config folder is etc). That is why i
> implemented it in casproperties.java.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/eurekaclinical/cas/pull/13/files/e8a03a461e680e5a354a323185d9a71e1520e508#r75984622>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AEyxJ4E6qLLcYgHMv-VcXiPdD_l9OjKzks5qi6ftgaJpZM4JqNQ3>
> .
>